icoscp/station/station.py

```diff
@@ -561,42 +562,147 @@ def get(stationId):
     #myStn._setData()        
     return myStn
 
-    
-def getIdList(project='ICOS', sort='name'):
-    """
-    Returns a list with all station id's. By default only only ICOS stations
-    will be returned. If filter is set to 'all',  all known station id's are retured.
-    Please be aware, that the usage of data associated with non-ICOS station
-    might be different then the CCBY 4.0 Licence at ICOS.
+
+def getIdList(project='ICOS', sort='name', outfmt='pandas'):
```

Comment:
there is a runtime error. if project='all', a list of all stations are returned, including ones with country code = UK.....the result from restcountries API however contains Great Britain 'GB' and not United Kingdom 'UK'

Reply:
This should be resolved now.